eventually/utils/validators.py

```diff
@@ -120,3 +121,21 @@ def list_of_int_validator(value):
     if not all(isinstance(item, int) for item in value):
         return False
     return True
+
+
+def password_validator(password):
+    """
+    Function that provides passwor validation.
+
+    Pasword should consist of: uppercase letters: A-Z, lowercase letters: a-z, numbers: 0-9
+
+    :param password: the password itself.
+    :type password: string
+
+    :return: `True` if value if valid and `False` if it is not.
+    """
+    if string_validator(password, min_length=4, max_length=32):
+        template = re.compile(r'^(?=.*?\d)(?=.*?[A-Z])(?=.*?[a-z])[A-Za-z\d]{4,32}$')
+        if template.match(password):
+            return True
+    return False
```

Comment:
This `if` statement is not necessary because if we return nothing from the function the result will be `none`.

Reply:
Okey)